### DISCUSSION TITLE
Payload Mount Documentation

### DISCUSSION THREAD

**Discussion by Maya-EG** [dannyjacobs/ECHO#51] [Q&A]
Hey, I'm a student at McGill trying to mount a Bicolog antenna onto our drone. There's several 3D print models of mounts posted on the website, but there's no information I can find about how the mounts come together, pros and cons of each design, etc. Does anyone have any documentation on how the mounts were constructed, or any insight regarding payload mount design? Please let me know, thanks!

**Comment by dannyjacobs** [comment]
The Modular mount is the one we're currently using.
https://github.com/dannyjacobs/ECHO/tree/master/hardware/STLs/11_Modular_Chiropter_Mount
The design idea is for a modular set of trays which connect together with all-thread. There are tweaks like extra flanges with slats to zip-tie cables or similar things.  We print the trays and use plastic all-thread/nuts to hold it all together.

**Reply by Maya-EG** [reply]
Perfect, thank you so much!
